app/Http/Resources/WeekScheduleResource.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Carbon\Carbon;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class WeekScheduleResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        $date = Carbon::now();
+        $date->setDate($this->year, $this->month, $this->day);
+
+        $status = null;
+
+        switch ($this->status) {
+            case 1: {
+                $status = config('api.status.short.' . $this->status);
+                break;
+            }
+            case 2: {
+                $status = config('api.status.short.' . $this->status);
+                break;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.ControlStructures.SwitchDeclaration.WrongOpenercase</br>**Severity**: error</br>**File**: app/Http/Resources/WeekScheduleResource.php</br>**Line**: 32</br>CASE statements must be defined using a colon